### PR TITLE
Djangocmstutorial

### PR DESCRIPTION
**Pull request by mebdivio**
## Related resources

* #...
* #...

## Checklist

* [ ] I have opened this pull request against ``develop``
* [ ] I have updated the **CHANGELOG.rst**
* [ ] I have added or modified the tests when changing logic

### CODE REVIEW

File: docs/how_to/custom_plugins.rst [hunk 1]
```diff
@@ -137,11 +137,11 @@ will require a model.
 The simplest plugin
 *******************
 
-You may use ``python manage.py startapp`` to set up the basic layout for you
+You may use ``python manage.py startapp`` to set up the basic layout for your
 plugin app (remember to add your plugin to ``INSTALLED_APPS``). Alternatively, just add a file called ``cms_plugins.py`` to an
 existing Django application.
 
-In ``cms_plugins.py``, you place your plugins. For our example, include the following code::
+Place your plugins in ``cms_plugins.py``. For our example, include the following code::
```
Comment (evildmp) [inline docs/how_to/custom_plugins.rst]: Please revert, original is preferred.

File: docs/introduction/09-third_party.rst [hunk 2]
```diff
@@ -39,37 +39,32 @@ to you to check them because you need to avoid duplication:
     'aldryn_common',
     'aldryn_newsblog',
     'aldryn_people',
-    'aldryn_reversion',
```
Comment (evildmp) [inline docs/introduction/09-third_party.rst]: Revert all changes in 09-third_party.rst, we will not make any changes here now.

Comment (mebdivio) [reply docs/introduction/09-third_party.rst]: aldryn_reversion file may not exist can you check it. For now, I left it. That is the one blocking it. Others are just duplicate conflicts.

File: docs/how_to/custom_plugins.rst (hunk 1 above)
Comment (mebdivio) [reply docs/how_to/custom_plugins.rst]: Done.

**Pull request merged**